packages/react-components/src/components/VideoGallery/styles/ScrollableHorizontalGallery.style.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStackStyles } from '@fluentui/react';
+import {
+  SMALL_HORIZONTAL_GALLERY_TILE_SIZE_REM,
+  SMALL_HORIZONTAL_GALLERY_TILE_STYLE
+} from './VideoGalleryResponsiveHorizontalGallery.styles';
+
+/**
+ * @private
+ */
+export const scrollableHorizontalGalleryStyles: IStackStyles = {
+  root: {
+    width: '100%',
+    minHeight: `${SMALL_HORIZONTAL_GALLERY_TILE_SIZE_REM.height}rem`,
+    overflow: 'scroll',
+    paddingRight: '0.5rem',
+    '-ms-overflow-style': 'none',
+    'scrollbar-width': 'none',
+    '::-webkit-scrollbar': { display: 'none' },
```

Comment:
concerned about this from an accessibility point of view - are there any a11y consequences to hiding the scrollbar or any a11y principles around scroll behavior? (I'll havea a google around this also and try find out, but would be worth asking the a11y team)

Reply:
So I got in touch with them and they are saying it required that it is visible. does Alex P have designs for what a scrollbar in the UI lib should look like?

Reply:
I will let Alex P know about your concerns and what the A11y team says. Thank you.